pages/linux/scriptreplay.md

```diff
@@ -0,0 +1,15 @@
+# scriptreplay
+
+> Output content of session files created by script command to standard output.
```

Comment:
```suggestion
> Replay a typescript created by the script command to standard output.
```
Seems clearer this way, what do you think?

Reply:
I like the edit. I also think I need to consider a remedial English grammar class. :\